rollbar/invites.go

```diff
@@ -17,13 +16,13 @@ type Invite struct {
 	DateRedeemed int    `json:"date_redeemed"`
 }
 
-// ListInvitesResponse : A data structure for the ListInvites response.
+// ListInvitesResponse represents the ListInvites response.
```

Comment:
these comments for structs are quite useless for me. They say the same as the names of the structs (which are greatly descriptive), but with spaces instead of camel case :)

Reply:
You have to include them, otherwise go will complain